lib/elixir/lib/access.ex

```diff
@@ -54,10 +54,7 @@ defmodule Access do
   end
 
   def fetch(list, key) when is_list(list) do
-    case :lists.keyfind(key, 1, list) do
-      {^key, value} -> {:ok, value}
-      false -> :error
-    end
+    Keyword.fetch(list, key)
```

Comment:
Not sure if we really want to "inline" this call since we already use `Keyword.get_and_update` below and I don't think Access calls are meant to be used with big lists (mostly small `options`, `params`, etc.).